regression/cbmc-concurrency/atomic_section_sc7/test-guard.desc

```diff
@@ -0,0 +1,12 @@
+KNOWNBUG
+main.c
+--program-only
+^EXIT=0$
+^SIGNAL=0$
+--
+^warning: ignoring
+__COUNT__#\d+ == (1 \+ __COUNT__#\d+|__COUNT__#\d+ \+ 1)
```

Comment:
Isn't the same expression in thr2 where it cannot be propagated?

Reply:
It actually can be propagated, because that entire function is an atomic section and the increment is guarded by `if(__COUNT__ == 1)` (in `thr2`, and `if(__COUNT__ == 0)` in `thr1`). As we nowadays constant-propagate such conditions, it ought to be fully constant.